app/Console/Commands/MigrateConfig.php

```diff
@@ -80,7 +80,12 @@ public function handle()
         // Get default values of legacy configuration variables.
         $legacy_defaults = [];
         $ONLY_LOAD_DEFAULTS = true;
-        include $cdash_app_dir . DIRECTORY_SEPARATOR . 'config' . DIRECTORY_SEPARATOR . 'config.php';
+        $config_dir = $cdash_app_dir . DIRECTORY_SEPARATOR . 'config' . DIRECTORY_SEPARATOR;
+        include $config_dir  . 'config.php';
+        if (file_exists($config_dir . 'config.local.php'))
+        {
+            include $config_dir . 'config.local.php';
+        }
```

Comment:
I believe the current CDash practice is to put the opening curly brace on the same line as the if-statement.  Only function and class declarations have the opening curly brace on a separate line.  The same change applies to the area around line 124 as well.

Our style checker didn't pick this up, so perhaps we should take a look at the rules being enforced to help standardize things like this.

Reply:
Thanks!